canonical-web-and-design/k8s.demo.haus
# List running demos on index page

## Done

List running demos on the index page with a link to them.

## QA

- dotrun
- in .env set `KUBECONFIG` to point at the demo cluster kubectl config file
- check `localhost:8100` to see the list of running demos
